index.html

```diff
@@ -11,9 +11,58 @@
   <body>
     <header>
       <nav>
-        <h3>About</h3>
         <h3>Skills & Work</h3>
+        <h3>About</h3>
+        <h3>Contact</h3>
       </nav>
     </header>
+
+    <main>
+      <h1>Welcome, I am Leonie</h1>
+      <h2>Nice to meet you!</h2>
+
+      <div>
+        <h2>Skills & Work</h2>
+        <p>Communication // Teamwork // Organisation</p>
+
+        <article>
```

Comment:
Since you have so many `article` elements as descendants here, I'd advise to use a `section` here, looks a bit more structured ;)